lib/modules/dosomething/dosomething_fact_page/dosomething_fact_page.module

```diff
@@ -90,22 +89,29 @@ function dosomething_fact_page_preprocess_node(&$vars) {
 
     foreach ($template_vars as $key => $labels) {
       foreach ($labels as $label) {
-        $field = "field_{$label}";
-          if (isset($content[$field])) {
+        $field = $vars["field_{$label}"];
+
+        if (isset($field) && !empty($field)) {
+          if (array_key_exists('en', $field)) {
+            $field = $field['en'];
```

Comment:
hmm, well you are correct they are not translatable, so all fact pages are currently in `en` but any new ones created would be `en-global` since that's the site default.